math/src/test/java/com/powsybl/math/matrix/AbstractMatrixTest.java

```diff
@@ -102,6 +102,36 @@ public void testMultiplication() {
         assertEquals(8, c.get(2, 0), EPSILON);
     }
 
+    @Test
+    public void testAddition() {
+        /*
+        1 0
+        0 3
+        2 0
+         */
+        Matrix a = createA(getMatrixFactory());
+        /*
+        4 0
+        5 0
+        0 6
+         */
+        Matrix b = getMatrixFactory().create(3, 2, 3);
+        b.set(0, 0, 4);
+        b.set(1, 0, 5);
+        b.set(2, 1, 6);
+        Matrix cs = a.add(b);
+        DenseMatrix c = cs.toDense();
```

Comment:
Maybe we could check that in the sparse representation we still have one "missing" value, therefore a values array of size 5 and not 6.

Reply:
Done